lib/ansible/modules/cloud/amazon/ec2_elb_lb.py

```diff
@@ -738,10 +738,24 @@ def _create_elb(self):
             self.status = 'created'
 
     def _create_elb_listeners(self, listeners):
-        """Takes a list of listener tuples and creates them"""
-        # True if succeeds, exception raised if not
-        self.changed = self.elb_conn.create_load_balancer_listeners(self.name,
-                                                                    complex_listeners=listeners)
+        """Takes a list of listener tuples and creates them
+
+        Arguments:
+            listeners (list): list of tuples
+
+        Returns:
+            True if succeeds
+
+        Raises:
+            IndexError: If HTTPS is specified as a listener but SSL cert is not specified.
```

Comment:
Technically these docs aren't quite right. This doesn't raise IndexError - it catches an IndexError and then calls `fail_json` itself, so callers don't need to also catch IndexError.

Reply:
Ahhh ok righto then, yeah now that you explain it like that I feel kinda stupid haha. Sorry about that, trying to make it clean has probably resulted in me trying to hard on this PR. So because it doesn't Raise the error, I should just remove the Raises section and leave the docstring as is? Or should I replace it with a Catch section? Sorry for the 50 questions here.